Local masvs creation (#534)
OWASP/owasp-masvs: README.md

Read the English version on [Gitbook](https://mobile-security.gitbook.io/masvs/ "GitBook Mobile AppSec Verification Standard"). The book is automatically synchronized with the main repo.

## Create new PDF, EPUB and Word document

> The steps were tested on macOS only!

> You can find the documents in the [release page](https://github.com/OWASP/owasp-masvs/releases). If you want to generate the documents yourself, execute the following steps:

Reviewer: ```suggestion
You can find the documents on the [release page](https://github.com/OWASP/owasp-masvs/releases). If you want to generate the documents yourself, execute the following steps:
```

> - The document creation is using a Docker container, so make sure that you have [Docker installed](https://www.docker.com/products/docker-desktop).

Reviewer: ```suggestion
- The document creation uses a Docker container, so make sure that you have [Docker installed](https://www.docker.com/products/docker-desktop).
```

- Install `gnu-sed` via brew. This will add the command `gsed` which is used to remove the comment from the `\pagebreak` LaTeX command.

```shell
$ brew install gnu-sed
```

- Clone the MASVS repository:

```shell
$ git clone https://github.com/OWASP/owasp-masvs/
```

- Create a new branch, as files will be changed when removing the comment from the `\pagebreak` LaTeX command:

```shell
$ cd owasp-masvs/
$ git checkout -b local_masvs_creation
```

- Run the document generation script for the chosen language:

```shell
$ ./tools/docker/pandoc_makedocs_local.sh Document-de LATEST
```

> - "Document-de" is specifying the folder of the language that is being used to generate the documents. Simply replace with the one you want to create.

Reviewer: ```suggestion
- "Document-de" specifies the folder of the language that is used to generate the documents. Simply replace it with the language you want to use.
```
